src/o365/spo/commands/app/app-remove.ts

```diff
@@ -173,12 +168,20 @@ class AppRemoveCommand extends SpoCommand {
       }
 
       if (args.options.appCatalogUrl) {
-        const isValidSharePointUrl: string | boolean = SpoCommand.isValidSharePointUrl(args.options.appCatalogUrl);
+        const isValidSharePointUrl: string | boolean = SpoAppBaseCommand.isValidSharePointUrl(args.options.appCatalogUrl);
```

Comment:
Since `isValidSharePointUrl` is defined in the `SpoCommand` class, it would be clearer to use that class, instead of `SpoAppBaseCommand`.